client/src/sagas/auth.js

```diff
@@ -0,0 +1,20 @@
+import { delay, put, takeLatest } from 'redux-saga/effects';
+import { LOGIN, LOGIN_ASYNC, REGISTER, REGISTER_ASYNC } from './types';
+
+export function* loginAsync({ payload }) {
+  yield delay(1000);
```

Comment:
For now I've just left these as dummy async functions, but these would be replaced with an axios request to the server